.github/workflows/run_functional_product_tests.yaml

```diff
@@ -0,0 +1,56 @@
+# Author: James Garriss
+# Purpose: Run the product functional tests every night
+
+name: Nightly Product Functional Tests
+
+# Run this workflow at midnight 
+# on every Sun to Thr (b/c GMT -5)
+on:
+  push:
+  schedule:
+  - cron: '0 4 * * 0-4'
+  workflow_dispatch: 
+
+jobs:
+  sharepoint-test:
+    name: SharePoint Test
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: powershell 
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
+    - name: Create PFX
+      run: |
+        New-Item -ItemType file -Path ./key.txt
+        Set-Content -Path ./key.txt -Value $env:PfxBase64
+        certutil -decode ./key.txt ./key.pfx
+      env:
+        PfxBase64: ${{ secrets.NIGHTLY_TEST_BUILD_PFX }}
+    - name: Import PFX
+      # Pipe to Out-Null to hide the thumbprint
+      run: Import-PfxCertificate -Password (ConvertTo-SecureString -String $env:PfxPassword -AsPlainText -Force) -CertStoreLocation Cert:\CurrentUser\My -FilePath ./key.pfx | Out-Null
+      env:
+        PfxPassword: ${{ secrets.NIGHTLY_TEST_BUILD_PW }}
+    - name: Get Thumbprint
+      id: thumbprint
+      run: |
+        $x509 = Get-PfxCertificate -FilePath ./key.pfx -Password (ConvertTo-SecureString -String $env:PfxPassword -AsPlainText -Force)
+        $Thumbprint = $x509.ThumbPrint
+        # Pass thumbprint to later job.
+        echo THUMBPRINT=$Thumbprint >> $env:GITHUB_OUTPUT
+      shell: pwsh
+      env:
+        PfxPassword: ${{ secrets.NIGHTLY_TEST_BUILD_PW }}
+    - name: Setup for Scuba
+      run: ./SetUp.ps1
+    - name: Install Selenium
+      run: Install-Module -Name Selenium
+    - name: Update Selenium
+      run: ./Testing/Functional/SmokeTest/UpdateSelenium.ps1
+    - name: Test SharePoint
+      run: |
+        # Read thumbprint from previous job.
+        $thumbprint = "${{ steps.thumbprint.outputs.THUMBPRINT }}"
+        ./Testing/Functional/Products/Tests/CallProductTests.ps1 $thumbprint
```

Comment:
Not very intuitive that CallProductTests will test "just" SharePoint.  What is the long term intent here?

Reply:
The comments in this script already address this:

```
The purpose of this script is to enable a GitHub Action workflow to
run the functional tests for all the products.
NOTE: At the moment it only does this for one product, Sharepoint.  Others
      will be added.
```

We are being agile, staring with one product and adding other later.  Issue 731, for example, will add Teams.